_config.yml

```diff
@@ -25,3 +25,7 @@ assets:
   sources:
     - "_assets/css"
     - "images"
+
+collections:
+    commands:
```

Comment:
This is spaced/tabbed differently than the rest of the file, causing things to not build/re-build/watch properly